### PR TITLE
Added comments to Sidebar and Container layouts

### PR DESCRIPTION
**Pull request by blackbaud-brandonhare**
Created a new partial for the comments. Did not update the blog pattern, as I didn't want to potentially break any existing stuff.

### CODE REVIEW

File: src/partials/partial-disqus.hbs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="comments">
+    <div id="disqus_thread"></div>
+        <script type="text/javascript">
+            (function() {
+            var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
+            dsq.src = '//{{ stache.config.comments_key }}.disqus.com/embed.js';
+            (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
+            })();
+        </script>
+    </div>
```
Comment (Blackbaud-SteveBrush) [inline src/partials/partial-disqus.hbs]: I think there's one too many closing `</div>` tags, here.

Comment (blackbaud-brandonhare) [reply src/partials/partial-disqus.hbs]: Two closing tags for two divs?

Comment (blackbaud-brandonhare) [reply src/partials/partial-disqus.hbs]: Ah crap, I see it.

File: src/layouts/layout-container.hbs [hunk 2]
```diff
@@ -11,6 +11,9 @@ layout: layout-base
         {{ else }}
           {{# withNewline }}{{ body }}{{/ withNewline }}
         {{/ inherit }}
+        {{# inherit stache.config.comments comments }}
```
Comment (Blackbaud-SteveBrush) [inline src/layouts/layout-container.hbs:14]: This is a boolean? If so, is it possible to use the boolean pattern from other boolean variables (ie, `showComments`)? If it's a pre-existing variable, don't worry about it.

Comment (blackbaud-brandonhare) [reply src/layouts/layout-container.hbs:14]: It's a preexisting variable (and it is a boolean). We use it in the blog pattern to enable comments there. It seemed silly to have two variables to enable comments in different places, so I just used that one.

Comment (Blackbaud-SteveBrush) [reply src/layouts/layout-container.hbs:14]: That's fine! I think this looks good.

**Comment by blackbaud-brandonhare** [comment]
@Blackbaud-BobbyEarl?

**Review by Blackbaud-BobbyEarl — CHANGES_REQUESTED** [review]
layout-sidebar inherits from layout-container.  In this proposed state, turning on the comments would mean the divs + scripts would show up twice for layout-sidebar.

layout-blog also inherits from layout-sidebar, so turning on comments would have them show up three times.

It would mean the comments spanning the content + sidebar, but perhaps it's best in layout-base?  That would mean all layouts are supported, but visually it may not look as good.  We would have to remove them from the blog pattern in that scenario.

**Comment by blackbaud-brandonhare** [comment]
@bobbyearl Yeah, I started out by adding it to base, but it looks terrible. I also didn't want to force people to turn off the comments on every page. I'd rather it be limited in scope. Is there a good way to restrict it to just the sidebar and container without the duplication?

**Comment by blackbaud-brandonhare** [comment]
Yeah, I think this makes more sense to do as a one off for the specific project that needs it. I'm reluctant to commit to the code churn required to properly implement it lest we break a bunch of other things. Closing this.

**Pull request closed without merging**